_config.yml

```diff
@@ -18,10 +18,10 @@
 # Feel free to change these! For more info, refer to:
 # ~> https://jekyllrb.com/docs/configuration/options/
 
-title:            'LA Housing'
+title:            'Divided Development'
 description:      'A Collection Enhancement Project'
 url:              'https://UCLALibrary.github.io'
-baseurl:          '/lahousing'
+baseurl:          '/divideddevelopment'
```

Comment:
this does not define the baseurl, rather it tells Jekyll what the baseurl is. For this site, the baseurl is `/lahousing` - changing this will break the site!